python-frontend/src/main/java/org/sonar/python/semantic/SymbolUtils.java

```diff
@@ -252,6 +251,42 @@ public static Optional<FunctionSymbol> getOverriddenMethod(FunctionSymbol functi
     return Optional.empty();
   }
 
+  public static Optional<FunctionSymbol> getFunctionSymbol(@Nullable Symbol symbol) {
```

Comment:
See my comment [here](https://github.com/SonarSource/sonar-python/pull/1559/files#r1323188284) as I reviewed this part in the second PR, which I happened to review before this one.